bin/main.rb

```diff
@@ -0,0 +1,59 @@
+# Set up game
+puts "Welcome!"
+puts "Player 1, What is your name?"
+@player1 = gets.chomp
+puts "Now, Player 2, What is your name?"
+@player2 = gets.chomp
+puts "Let's play! #{@player1} #{@player2}"
+continue_game = true
+
+# While nobody has won
+while continue_game
+  # get input from player 1
+  puts "Choose a number between 1-9 #{@player1}"
+  @player1_input = gets.chomp.to_i
+  # Make sure user has the correct input
+  if @player1_input.is_a?(Integer) && @player1_input > 0 && @player1_input <= 9
+    print "Player one, you have input #{@player1_input}"
+    puts
+    puts "Have you won? press 'y' for yes and 'n' for no 't' for tie"
+    ans = gets.chomp
+    if ans == 'y'
+      continue_game = false
+      break
+    elsif ans == 't'
+      puts 'IT IS A TIE'
+      break
+    end
+  else
+      # If player 1 has wrong input
+    puts "YOU GOTTA PUT IN SOMETHING RIGHT"
+  end
+
+  puts "HERE IS YOUR BOARD RIGHT HERE."
+
+  puts "Choose a number between 1-9 #{@player2}"
+  @player2_input = gets.chomp.to_i
+  if @player2_input.is_a?(Integer) && @player2_input > 0 && @player2_input <= 9
+    puts "#{@player2} now its your turn, choose a number between 1-9"
+    puts
+    puts "Have you won? press 'y' for yes and 'n' for no 't' for tie"
+    ans = gets.chomp
+    if ans == 'y'
+      continue_game = false
+      break
+    elsif ans == 't'
+      puts 'IT IS A TIE'
+      break
+    end
+  else
+    # If player 2 has wrong input
+    puts "FOLLOW THE RULES"
+  end
+
+  puts "HERE IS YOUR BOARD RIGHT HERE."
```

Comment:
Where is it please ?

Reply:
If you take a look at the Milestone 3, you can see that we have a board. I made the personal decision that this milestone was to prove we had the basic idea for this game and the flow, therefore not seeing the necessity to display a board.

Reply:
Especially since this is just a basic "input/output" stage of the project. I really don't see the necessary requirements to create a board, but to merely SHOW that we know where it should show.

Reply:
In fact, if you take a look at the project outline itself for Milestone 2, it clearly states, and I quote **_"Display prompts for users (hint: puts/print), collect user input (hint: get.chomp) and only mock the basic flow of the game (you can use puts or inline comments to say for example “Now, your move is displayed on the board”)."_**